### PR TITLE
Update CONTRIBUTING.md to point to the How to contribute page

### PR DESCRIPTION
**Pull request by axelsimon**
Moving things around so we eventually have:
enarx/enarx.github.io/CONTRIBUTING.md --> https://github.com/enarx/enarx/wiki/How-to-contribute
enarx/enarx/CONTRIBUTING.md --> https://github.com/enarx/enarx/wiki/How-to-contribute

Companion PR to https://github.com/enarx/enarx.github.io/pull/23.

### CODE REVIEW

**Comment by lkatalin** [comment]
Is the conference template supposed to be part of this same PR?

**Comment by mbestavros** [comment]
@lkatalin No, this is just a link update in our documentation.

File: .github/ISSUE_TEMPLATE/conferences.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- CFP closes:    DD – DD Month YYYY
+- CFP results:   DD – DD Month YYYY
+- Slides due:    DD – DD Month YYYY
+- Event:         DD – DD Month YYYY
+
+**Location**
+Venue, City, (State,) Country
+
+**CFP**
+URL to the Call for Proposals / Papers:
```
Comment (mbestavros) [inline .github/ISSUE_TEMPLATE/conferences.md]: This shouldn't be included with this PR.

**Comment by axelsimon** [comment]
Indeed, that's a mistake. I'll correct first thing tomorrow.
Another opportunity to get better at git ;)

**Comment by axelsimon** [comment]
I ended up in a strange situation where `git status` kept telling me:
```
Your branch and 'origin/move_CONTRIBUTING_to_main_repo' have diverged,                                                                  
and have 18 and 2 different commits each, respectively.
  (use "git pull" to merge the remote branch into yours)
```
even after rebasing on `upstream/master`. I eventually forced push my local branch, which closed this PR as there where no more commits, recreated the change, the commit, and pushed. Things are now in a state where there is only one commit and one change, the one we wanted.

Probably not the most elegant solution, but we are ready to go i think.

**Pull request merged**